operator/tools/go.mod

```diff
@@ -0,0 +1,73 @@
+module github.com/stackrox/stackrox/operator/tools
+
+go 1.18
```

Comment:
What would be needed to use go 1.19 ?

Reply:
It's a long story. @RTann started this process. we need to update Go in apollo image, then fix formatting to pass checks, then update image in here and osci. I hope we can skip builds and unit tests on osci soon and keep it only for e2e so bumping go version will be easier. 
See #4589